ITE/9/README.adoc

```diff
@@ -59,6 +59,10 @@ The new attestation definition SHOULD include the following fields.
 
 This section must discuss at a high level the reason for creating the new attestation type. Essentially, it describes the problems that can be solved using the new attestation.
 
+==== Use case
+
+This section must discuss the use case that will be solved by this predicate and how the current predicate types (if any) do not solve this.
```

Comment:
I think "solved" is probably too strong a word here (but also adding a little more concreteness).

```suggestion
This section must discuss at least one concrete use case that will be covered by this predicate and how the current predicate types (if any) do not solve this.
```